content/general/concepts/function/function.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'Functions'
+Description: 'The function data type.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Data Types'
+  - 'Types'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+Depending on the language, functions can be utilized as variables and arguments for other functions.
+
+```javascript
+function someFunction() {
+  return 'Hello World!';
+} // nothing happends
+
+somefunction(); // outputs Hello World! because the function was called
+
+function otherFunction(name) {
+  return `Hi ${name} welcome to Codecademy docs!`;
+} // nothing happends
+
+otherFunction('Codecademy'); // outputs Hi Codecademy welcome to Codecademy docs! because the function was called and we use the Codecademy as the agument.
+```
```

Comment:
```suggestion
## Functions in Different Languages

* [C: Functions](https://www.codecademy.com/resources/docs/c/functions)
* [C++: Functions](https://www.codecademy.com/resources/docs/cpp/functions)
* [JavaScript: Functions](https://www.codecademy.com/resources/docs/javascript/functions)
* [PHP: Functions](https://www.codecademy.com/resources/docs/php/functions)
* [Python: Functions](https://www.codecademy.com/resources/docs/python/functions)
* [Ruby: Functions](https://www.codecademy.com/resources/docs/ruby/functions)
* [TypeScript: Functions](https://www.codecademy.com/resources/docs/typescript/functions)
```

Instead of using a language-specific snippet for a General Entry, this would be a great opportunity to spotlight some language-specific entries about functions. Following your link strategy from [General: Data Types](https://www.codecademy.com/resources/docs/general/data-types) 😄